### PR TITLE
Sentry update and timeout warning

### PR DESCRIPTION
**Pull request by kalilsn**
#2455 

This PR has a few small updates to our sentry config intended to get some insight into these timeout errors (which are coming from the heroku router, and therefore not captured by sentry).

- Updates to latest sentry versions
- Switch to sentry react package over sentry browser
- Adds a message to sentry when no response is sent after 25s
- Move frontend to its own sentry project

### CODE REVIEW

File: client/components/Editor/plugins/malformedDoc.ts [hunk 1]
```diff
@@ -60,15 +60,8 @@ export default () => {
...
-					try {
-						throw new Error('Uh-oh!');
-					} catch (err: any) {
-						theError = err;
-					}
-					Sentry.captureEvent({
-						message: 'Evil Prosemirror Transaction',
-						contexts: { transactionMetadata, error: theError },
+					Sentry.captureException(new Error('Evil Prosemirror Transaction'), {
+						contexts: { transactionMetadata },
```
Comment (kalilsn) [inline client/components/Editor/plugins/malformedDoc.ts:64]: This refactor was to fix a type error. I think it's mostly equivalent but let me know if I misunderstood the goal of this block!

Comment (idreyn) [reply client/components/Editor/plugins/malformedDoc.ts:64]: No, thank you for removing this ridiculous little bit of code!

**Review by idreyn — APPROVED** [review]
LGTM and I'm excited to get closer to the heart of the timeout mystery!

File: client/components/Editor/plugins/malformedDoc.ts (hunk 1 above)
Comment (howebenhowe) [reply client/components/Editor/plugins/malformedDoc.ts:64]: I have re-read the previous implementation here a few times and can't reason why there was a try-catch block there. It reads like a code-koan.
"Try to fail, and if you do, you have your answer, so continue."

**Review by howebenhowe — APPROVED** [review]
LGTM!

**Review by qweliant — APPROVED** [review]
LGTM

**Pull request merged**